lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -396,7 +422,15 @@ export default function ContentSelector({
             Canvas Page
           </OptionButton>
         )}
-
+        {canvasStudioEnabled && (
+          <OptionButton
+            data-testid="canvas-studio-button"
+            details="Canvas Studio"
```

Comment:
Related to this, we need a video/media icon for the file picker. Not for this PR of course.

Reply:
I've added a follow-up task on the Canvas Studio board to update icons and terminology to be video-centric.